package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "prettier-plugin-sort-imports",
-	"version": "1.7.1",
+	"version": "1.7.2",
```

Comment:
Let's not make this change yet. I'll submit that as a separate patch after yours is merged :)